plugins/source/alicloud/go.mod

```diff
@@ -0,0 +1,47 @@
+module github.com/cloudquery/cloudquery/plugins/source/alicloud
+
+go 1.19
+
+require (
+	github.com/alibabacloud-go/tea v1.1.20
+	github.com/aliyun/alibaba-cloud-sdk-go v1.62.80
+	github.com/aliyun/aliyun-oss-go-sdk v2.2.6+incompatible
+	github.com/cloudquery/plugin-sdk v1.11.1
```

Comment:
This plugin-sdk version is a little out-of-date, let's make sure to use the latest version (`v1.21.0`)